submissions/Shizopo/html-css-popup/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="./styles/index.css">
+    <title>Wooble</title>
+</head>
+<body>
+    <div class="header">
+        <a href="#" class="header-link" title="Your light and secure mail">Wmail</a>
+        <a href="#" class="header-link" title="Your photo gallery">Photos</a>
```

Comment:
Use the special `<header>` tag to display the menu

Reply:
Done